dataactvalidator/config/awardFields.csv

```diff
@@ -1,61 +1,61 @@
-cfda_title,cfda_title,FALSE,str,100,,
-correctionlatedeleteindicator,correction_late_delete_ind,FALSE,str,1,D1,
-facevalueloanguarantee,face_value_loan_guarantee,FALSE,float,20,D21,
-fain,fain,FALSE,str,30,"C9, D2, D5, D6",
-federalactionobligation,federal_action_obligation,FALSE,float,20,"C12, D20",
-fiscalyearandquartercorrection,fiscal_year_and_quarter_co,FALSE,str,5,D8,
-fundingagencycode,funding_agency_code,FALSE,str,3,D1,
-fundingagencyname,funding_agency_name,FALSE,str,100,,
-fundingagencyofficename,funding_office_name,FALSE,str,100,,
-fundingofficecode,funding_office_code,FALSE,str,3,,
-fundingsubtieragencycode,funding_sub_tier_agency_co,FALSE,str,4,D1,
-fundingsubtieragencyname,funding_sub_tier_agency_na,FALSE,str,100,,
-legalentityaddressline1,legal_entity_address_line1,FALSE,str,150,D2,
-legalentityaddressline2,legal_entity_address_line2,FALSE,str,150,,
-legalentityaddressline3,legal_entity_address_line3,FALSE,str,35,,
-legalentitycitycode,legal_entity_city_code,FALSE,str,5,,
-legalentitycityname,legal_entity_city_name,FALSE,str,40,,
-legalentitycongressionaldistrict,legal_entity_congressional,FALSE,str,2,,
-legalentitycountrycode,legal_entity_country_code,FALSE,str,3,"D1, D2, D12",
-legalentitycountycode,legal_entity_county_code,FALSE,str,3,,
-legalentitycountyname,legal_entity_county_name,FALSE,str,40,,
-legalentityforeigncityname,legal_entity_foreign_city,FALSE,str,40,D9,
-legalentityforeignpostalcode,legal_entity_foreign_posta,FALSE,str,50,,
-legalentityforeignprovincename,legal_entity_foreign_provi,FALSE,str,25,,
-legalentitystatecode,legal_entity_state_code,FALSE,str,2,,
-legalentitystatename,legal_entity_state_name,FALSE,str,50,,
-legalentityzip5,legal_entity_zip5,FALSE,str,5,"D2, D11",
-legalentityziplast4,legal_entity_zip_last4,FALSE,str,4,"D2, D11",
-nonfederalfundingamount,non_federal_funding_amount,FALSE,float,20,,
-originalloansubsidycost,original_loan_subsidy_cost,FALSE,float,20,D21,
-periodofperformancecurrentenddate,period_of_performance_curr,FALSE,str,8,"D3, D4, D19",
-periodofperformancestartdate,period_of_performance_star,FALSE,str,8,"D3, D18",
-primaryplaceofperformancecityname,place_of_performance_city,FALSE,str,100,,
-primaryplaceofperformancecode,place_of_performance_code,TRUE,str,7,"D1, D14, D15",
-primaryplaceofperformancecongressionaldistrict,place_of_performance_congr,FALSE,str,2,,
-primaryplaceofperformancecountrycode,place_of_perform_country_c,TRUE,str,3,D1,
-primaryplaceofperformancecountyname,place_of_perform_county_na,FALSE,str,40,,
-primaryplaceofperformanceforeignlocationdescription,place_of_performance_forei,FALSE,str,100,D17,
-primaryplaceofperformancestatename,place_of_perform_state_nam,FALSE,str,50,,
-primaryplaceofperformancezip_4,place_of_performance_zip4a,FALSE,str,10,"D2, D16",
-recordtype,record_type,TRUE,int,1,D1,
-sai_number,sai_number,FALSE,str,50,,
-totalfundingamount,total_funding_amount,FALSE,float,20,,
-uri,uri,FALSE,str,70,"C9, D6, D7",
+actiondate,action_date,FALSE,str,,"D3, D4",
```

Comment:
lazy question w/o looking at the downstream code--what does it mean when the field length is left blank?